### ISSUE TITLE
Isotope backend modules improvement

### ISSUE THREAD

**Issue by qzminski** [isotope/core#1544]
https://github.com/isotope/core/blob/master/system/modules/isotope/config/config.php#L152-L165

Don't you think this code should be initialized in the initializeSystem HOOK after all modules are loaded?

At the moment I need to register table in two places:

```php
$GLOBALS['BE_MOD']['isotope']['iso_setup']['tables'][] = 'tl_iso_my_table';
$GLOBALS['ISO_MOD']['config']['configs']['tables'][] = 'tl_iso_my_table';
```

**Comment by aschempp** [comment]
@qzminski wanna check this out?

**Comment by aschempp** [comment]
Implemented in #1624
